doc/tutorial.md

```diff
@@ -1,3 +1,259 @@
-# IAMF tutorial
+###Constructing A One-Region Model
+---
 
-TODO
+In this example, we will construct a stylized model of the global economy and its changing greenhouse gas emission levels through time. The overall strategy will involve creating components for the economy and emissions separately, and then defining a model where the two components are coupled together.  
+
+There are two main steps to creating a component:
+- [ ] Define the component using ``@defcomp`` where the parameters and variables are listed.
+- [ ] Use the timestep function ``timestep(state::component_name, t:Int)`` to set the equations of that component.  
```

Comment:
Hey David,

I had initially uploaded the text to a test .md document on bit bucket and everything was showing up as being formatted correctly. It also has a preview feature to see how your text will appear on bit bucket so you can make and check changes quickly. The [ ] gave a square bullet point in my version.

Best,
Frank

Sent from my iPhone

> On Jul 3, 2015, at 12:21 PM, David Anthoff <notifications@github.com> wrote:
> 
> In doc/tutorial.md:
> 
> >  
> > -TODO
> > +In this example, we will construct a stylized model of the global economy and its changing greenhouse gas emission levels through time. The overall strategy will involve creating components for the economy and emissions separately, and then defining a model where the two components are coupled together.  
> > +
> > +There are two main steps to creating a component:
> > +- [ ] Define the component using ``@defcomp`` where the parameters and variables are listed.
> > +- [ ] Use the timestep function ``timestep(state::component_name, t:Int)`` to set the equations of that component.  
> I think this is not the right syntax for a markdown list. Probably enough to just remove the [ ]
> 
> —
> Reply to this email directly or view it on GitHub.
>